public/app/plugins/datasource/prometheus/querybuilder/shared/parsingUtils.ts

```diff
@@ -2,8 +2,8 @@ import { SyntaxNode, TreeCursor } from '@lezer/common';
 
 import { QueryBuilderOperation } from './types';
 
-// This is used for error type for some reason
-export const ErrorName = '⚠';
+// Although 0 isn't explicitly provided in the lezer-promql library as the error node ID, it does appear to be the ID of error nodes within lezer.
```

Comment:
I'd mentioned here both `lezer-promql` and `@grafana/lezer-logql`: 

`... in the lezer-promql and @grafana/lezer-logql libraries as the error node ID...`